### PR TITLE
[kesch] ImageMagick and Ghostscript

### PR DESCRIPTION
**Pull request by lucamar**
EasyBuild configuration file of the latest stable release of `ImageMagick`and of its dependency `Ghostscript`.  I still could not update the dependency `freetype`to last release `2.8.1`.

### CODE REVIEW

**Comment by gppezzi** [comment]
Could you please also add another ImageMagick recipe for version 6.7 (current version in production) ? 

The plan is to install both versions under `experimental`, so that they can validate & decide which will go to the new production suite.

**Comment by lucamar** [comment]
I have included in the pull request the release `6.7.6-10`, available on the past releases of the official download webpage of ÌmageMagick`.

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.22-gmvolf-17.02.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+name = 'Ghostscript'
+version = '9.22'
+downloadversion = version.replace('.','')
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'gmvolf', 'version': '17.02'}
```
Comment (gppezzi) [inline easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.22-gmvolf-17.02.eb]: As discussed, please use either `GCC` or `dummy` for this PR

Comment (lucamar) [reply easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.22-gmvolf-17.02.eb]: Updated recipes with `dummy` toolchain

**Comment by lucamar** [comment]
Current issues:
 - `libjpeg-turbo` dependency commented out in `dummy` as could not build successfully using `NASM`
 - forced to rollback to `freetype/2.7.1` with `Ghostscript` in `gmvolf` to avoid version conflict of `zlib` (`1.2.11` vs. `1.2.8` used to build the toolchain `gmvolf/17.02`)

**Comment by lucamar** [comment]
I have updated the `ImageMagick` recipes using the tollchain `GCC-5.4.0-2.26.eb`, including a fix of `bzip2` recipes in order to create the `pkgconfig` folder used by the `pkg-config` command.

**Comment by lucamar** [comment]
The builds were successful on KeschTDS:
https://jenkins.cscs.ch/job/TestingEB/label=keschTDS/820/console
The project `TestingEB` failed on Monch, but this failure unrelated to the current pull request::
`java.lang.NoClassDefFoundError: Could not initialize class jenkins.model.Jenkins$MasterComputer`

**Pull request merged**